apps/condo/domains/acquiring/schema/RegisterMultiPaymentForInvoicesService.test.js

```diff
@@ -336,6 +343,72 @@ describe('RegisterMultiPaymentForInvoicesService', () => {
                 }),
             ]))
         })
+
+        test('register multiPayment for invoice from different organization', async () => {
+            const utils1 = new TestUtils([ResidentTestMixin])
+            const utils2 = new TestUtils([ResidentTestMixin])
+            await utils1.init()
+            await utils2.init()
+
+            await utils2.updateAcquiringContext({
+                invoiceStatus: CONTEXT_FINISHED_STATUS,
+                invoiceImplicitFeeDistributionSchema: [{
+                    recipient: 'organization',
+                    percent: '5',
+                }],
+                invoiceRecipient: createTestRecipient(),
+            })
+
+            const resident = await utils1.createResident()
+
+            const [marketCategory] = await createTestMarketCategory(utils2.clients.admin)
+            const [marketItem] = await createTestMarketItem(utils2.clients.admin, marketCategory, utils2.organization)
+            const [itemPrice] = await createTestMarketItemPrice(utils2.clients.admin, marketItem)
+            const [priceScope] = await createTestMarketPriceScope(utils2.clients.admin, itemPrice, utils2.property)
+
+            let [invoice] = await registerResidentInvoiceByTestClient(
+                utils1.clients.resident,
+                pick(resident, 'id'),
+                [{
+                    priceScope: pick(priceScope, 'id'),
+                    count: 1,
+                }],
+            );
+
+            [invoice] = await updateTestInvoice(utils2.clients.admin, invoice.id, { status: INVOICE_STATUS_PUBLISHED })
+
```

Comment:
Please, add TODO to test express route handler for this mutation. There are no tests for it
in format 
// TODO(userName): TASK-NUMBER and some description